src/detectors/B0TRK/B0TRK.cc

```diff
@@ -26,15 +24,10 @@ void InitPlugin(JApplication *app) {
     SiliconTrackerDigiConfig digi_default_cfg;
     digi_default_cfg.threshold = 0 * dd4hep::keV;
     digi_default_cfg.timeResolution = 8;
-    app->Add(new JChainFactoryGeneratorT<SiliconTrackerDigi_factory>({"B0TrackerHits"}, "B0TrackerHitRawHit", digi_default_cfg));
+    app->Add(new JChainFactoryGeneratorT<SiliconTrackerDigi_factory>({"B0TrackerHits"}, "B0TrackerRawHits", digi_default_cfg));
 
     // Convert raw digitized hits into hits with geometry info (ready for tracking)
-    TrackerHitReconstructionConfig hit_reco_cfg;
-    hit_reco_cfg.time_resolution = 8;
-    app->Add(new JChainFactoryGeneratorT<TrackerHitReconstruction_factory>(
-            {"B0TrackerHitRawHit"},     // Input data collection tags
-            "B0TrackerRecHits",         // Output data tag
-             hit_reco_cfg));            // Hit reco default config for factories
+    app->Add(new JChainFactoryGeneratorT<TrackerHitReconstruction_factory>({"B0TrackerRawHits"},"B0TrackerRecHits"));
```

Comment:
I'd keep this config option even if not used now.

Reply:
You mean to keep the hit_reco_cfg option in there with the time resolution?

Reply:
Done